frontend/device/generic/powerd.lua

```diff
@@ -219,6 +219,9 @@ function BasePowerD:getCapacity()
         now_ts = TimeVal:now()
     end
 
+    -- Add time the device was in standby
+    now_ts = now_ts + self.device.total_standby_tv
+
     if (now_ts - self.last_capacity_pull_time):tonumber() >= 60 then
         self.batt_capacity = self:getCapacityHW()
         self.last_capacity_pull_time = now_ts
```

Comment:
I don't want to know much about how all this work.
But just reading these lines, they feel very odd:
- assuming `self.device.total_standby_tv` is true to its name (the cumulative durations spent in standby), it's something that will only increase, so at some point reaching a few minute or hours.
- you add it to `now_ts`, which is saved as `self.last_capacity_pull_time` , and these will soon become timestamps in the future - so no more true to their names
- it might give you the expected results, but it feels odd :)

Reply:
Yes, but this is the same way it is done in autosuspend. 

We have some clocks running in KOReader:
 * `monotonic_coarse` for UIManager (and most other parts)
 * `boottime_or_realtime_coarse` for summing up standby times.
 
Would naming `now_tv` to something like `now_boottime_tv` make it less odd? here (and in autosuspend)?

I am on the hunt, if there are other similar things waiting constructs in KOReader.

Reply:
> Would naming now_tv to something like now_boottime_tv make it less odd? 

Possibly. But let's wait for @NiLuJe thoughts (he is the Master of Time, I'm just an end user :)

Reply:
@NiLuJe 
Btw, `now_ts` in getCapacity seem odd by itself. Shouldn't it be `now_tv`?
`ts` for time in secs, `tv` for timeval?

Reply:
Good point, +1!

Reply:
> > Would naming now_tv to something like now_boottime_tv make it less odd? 
> 
> Possibly. But let's wait for @NiLuJe thoughts (he is the Master of Time, I'm just an end user :)

I'll try to give it a look tonight, otherwise on Saturday.